docs/source/03_tutorial/05_namespacing_pipelines.md

```diff
@@ -0,0 +1,230 @@
+# Namespacing pipelines
+
+This section covers the following:
+
+* A brief introduction to namespaces and modular pipelines
+* How to convert the existing spaceflights project into a namespaced one
+* This pattern is viewed as the modern way to use Kedro after being first introduced in version 0.17.0
```

Comment:
It makes sense to add this sentence in this section, but it's not really one of the points being covered as far as I can tell. Perhaps just move it the next paragraph?